include/godot_cpp/core/class_db.hpp

```diff
@@ -184,6 +183,7 @@ void ClassDB::_register_class(bool p_virtual) {
 		T::property_can_revert_bind, // GDExtensionClassPropertyCanRevert property_can_revert_func;
 		T::property_get_revert_bind, // GDExtensionClassPropertyGetRevert property_get_revert_func;
 		T::notification_bind, // GDExtensionClassNotification notification_func;
+		T::notification_bind2, // GDExtensionClassNotification2 notification_func;
```

Comment:
We shouldn't be passing both of these. Since godot-cpp is being updated with awareness of `classdb_register_extension_class2`, it should just pass `nullptr` for the old function pointer.